ice/classes/sanger_analysis.py

```diff
@@ -139,6 +139,7 @@ def initialize_with(self, control_path, edited_path, gRNA_sequences,
             gRNA_sequences = gRNA_sequences.split(",")
             gRNA_sequences = list(map(str.strip, gRNA_sequences))
             gRNA_sequences = list(map(RNA2DNA, gRNA_sequences))
+            gRNA_sequences = list(map(lambda x:x.upper(),gRNA_sequences))
```

Comment:
Doesn't really matter, but my linter always complains when there are missing spaces that don't follow PEP 8 conventions like between 'upper(),gRNA'.